### PR TITLE
Format transactions row and details

### LINKED ISSUE #561: Issue with double minus showing up on mixed transactions.
Double minus sign appears to be from mixed tx types.

Amount: --0.0001843 DCR, Tx direction: 0, Tx Type: Mixed
Amount: --0.0003251 DCR, Tx direction: 0, Tx Type: Mixed
Amount: --0.0000542 DCR, Tx direction: 0, Tx Type: Mixed
image

### PR DESCRIPTION
**Pull request by beansgum**
Closes #561 

- Fix minus sign not showing on some sent transactions

- Fix transferred transactions showing as sent

- Add mixed transaction filter to dropdown selector

- Format mixed transaction details

- Format stake transaction row

- Format vote transaction details

### CODE REVIEW

File: ui/page/components/components.go [hunk 1]
```diff
@@ -69,47 +68,52 @@ func UniformPadding(gtx layout.Context, body layout.Widget) layout.Dimensions {
...
-	if len(balanceParts) == 1 {
-		isDecimal = false
-		balanceParts = strings.Split(balance, " ")
+func TransactionTitleIcon(l *load.Load, wal *dcrlibwallet.Wallet, tx *dcrlibwallet.Transaction) (string, *widget.Image) {
+	var title string
+	var icon *widget.Image
+
+	if tx.Type == dcrlibwallet.TxTypeRegular {
+		if tx.Direction == dcrlibwallet.TxDirectionSent {
+			title = "Sent"
```
Comment (oshorefueled) [inline ui/page/components/components.go:77]: Can the texts here be added to the strings file so language translation can be supported?

File: ui/load/load.go [hunk 2]
```diff
@@ -40,14 +40,14 @@ type Icons struct {
...
 	PendingIcon, Logo, RedirectIcon, ConfirmIcon, NewWalletIcon, WalletAlertIcon,
-	ImportedAccountIcon, AccountIcon, EditIcon, expandIcon, CopyIcon, mixer, MixerSmall,
-	ArrowForwardIcon, TransactionFingerPrintIcon, SettingsIcon, SecurityIcon, HelpIcon,
+	ImportedAccountIcon, AccountIcon, EditIcon, expandIcon, CopyIcon, MixedTx, mixer, MixerSmall,
+	ArrowForwardIcon, Next, TransactionFingerPrintIcon, SettingsIcon, SecurityIcon, HelpIcon,
 	AboutIcon, DebugIcon, VerifyMessageIcon, LocationPinIcon, AlertGray, ArrowDownIcon,
 	WatchOnlyWalletIcon, CurrencySwapIcon, SyncingIcon, ProposalIconActive, ProposalIconInactive,
 	Restore, DocumentationIcon, DownloadIcon, TimerIcon, TicketIcon, TicketIconInactive, StakeyIcon,
-	List, ListGridIcon, DecredSymbolIcon *widget.Image
+	List, ListGridIcon, DecredSymbolIcon, DecredSymbol2 *widget.Image
```
Comment (Sirmorrison) [inline ui/load/load.go:50]: Rename `DecredSymbol2` to `DecredSymbolIconTwo`

File: ui/load/load.go [hunk 3]
```diff
@@ -40,14 +40,14 @@ type Icons struct {
 	ImageBrightness1, ContentClear, DropDownIcon, Cached, ContentRemove *widget.Icon
 
 	OverviewIcon, OverviewIconInactive, WalletIcon, WalletIconInactive,
-	ReceiveIcon, TransactionIcon, TransactionIconInactive, SendIcon, MoreIcon, MoreIconInactive,
+	ReceiveIcon, Transferred, TransactionIcon, TransactionIconInactive, SendIcon, MoreIcon, MoreIconInactive,
 	PendingIcon, Logo, RedirectIcon, ConfirmIcon, NewWalletIcon, WalletAlertIcon,
-	ImportedAccountIcon, AccountIcon, EditIcon, expandIcon, CopyIcon, mixer, MixerSmall,
-	ArrowForwardIcon, TransactionFingerPrintIcon, SettingsIcon, SecurityIcon, HelpIcon,
+	ImportedAccountIcon, AccountIcon, EditIcon, expandIcon, CopyIcon, MixedTx, mixer, MixerSmall,
+	ArrowForwardIcon, Next, TransactionFingerPrintIcon, SettingsIcon, SecurityIcon, HelpIcon,
```
Comment (Sirmorrison) [inline ui/load/load.go:46]: `Next` to `NextIcon`

File: ui/load/load.go [hunk 4]
```diff
@@ -40,14 +40,14 @@ type Icons struct {
 	ImageBrightness1, ContentClear, DropDownIcon, Cached, ContentRemove *widget.Icon
 
 	OverviewIcon, OverviewIconInactive, WalletIcon, WalletIconInactive,
-	ReceiveIcon, TransactionIcon, TransactionIconInactive, SendIcon, MoreIcon, MoreIconInactive,
+	ReceiveIcon, Transferred, TransactionIcon, TransactionIconInactive, SendIcon, MoreIcon, MoreIconInactive,
```
Comment (Sirmorrison) [inline ui/load/load.go:43]: `Transferred` to `TransferredIcon`

**Review by Sirmorrison — CHANGES_REQUESTED** [review]
image

The ticket type/direction should not be on the tx list page the Icon is enough. It should be on the Tx details page.

**Review by oshorefueled — CHANGES_REQUESTED** [review]

File: ui/page/transaction_details_page.go [hunk 5]
```diff
@@ -464,14 +656,11 @@ func (pg *TransactionDetailsPage) viewTxn(gtx layout.Context) layout.Dimensions
...
-	m := values.MarginPadding20
-	mtb := values.MarginPadding5
-	return layout.Inset{Left: m, Right: m, Top: mtb, Bottom: mtb}.Layout(gtx, body)
+	return layout.UniformInset(values.MarginPadding16).Layout(gtx, body)
 }
 
 func (pg *TransactionDetailsPage) separator(gtx layout.Context) layout.Dimensions {
-	m := values.MarginPadding5
-	return layout.Inset{Top: m, Bottom: m}.Layout(gtx, pg.theme.Separator().Layout)
+	return pg.theme.Separator().Layout(gtx)
```
Comment (oshorefueled) [inline ui/page/transaction_details_page.go]: Is this separator wrapper needed?

**Review by oshorefueled — CHANGES_REQUESTED** [review]

File: ui/load/load.go (hunk 3 above)
Comment (beansgum) [reply ui/load/load.go:46]: It's redundant to have `Icon` in the variable since the struct name already contains that.

**Review by oshorefueled — APPROVED** [review]

**Review by Sirmorrison — APPROVED** [review]

**Pull request merged**